analysis_service/base/migrations/0001_initial.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
```

Comment:
Before we set the generic name `base` into stone using the migration system, I'd strongly suggest to split the app into multiple apps `clusters`, `workers`, `spark_jobs` and a have the common code in the `analysis_service` package. In my experience having every domain objects in one package will eventually make maintenance much harder and adding/removing features risky. I'm happy to work on a separate PR after this got merged if you're okay with that pattern.